Examples-Web/AnimatedProgressButtons/CircularProgress/CircularProgressButtons.html

```diff
@@ -0,0 +1,29 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Animated Circular Progress Button Example</title>
+    <script name="TouchFree_Tooling" type="text/javascript" src="../TouchFree/TouchFree_Tooling.js"></script>
+    <script type="text/javascript" src="./CircularProgress.js"></script>
+    <link rel="stylesheet" href="../Progress.css">
+    <link rel="stylesheet" href="./CircularProgress.css">
+</head>
+
+<body>
+    <div class="container">
+        <button class="button" style="background-color:var(--raspberry-sorbet);"></button>
+        <button class="button" style="background-color:var(--burnt-coral);"></button>
+        <button class="button" style="background-color:var(--marigold);"></button>
+        <button class="button" style="background-color:var(--illuminating);"></button>
+    </div>
+    <div class="container">
+        <button class="button" style="background-color:var(--mint);"></button>
+        <button class="button" style="background-color:var(--green-ash);"></button>
+        <button class="button" style="background-color:var(--cerulean);"></button>
+        <button class="button" style="background-color:var(--amythist-orchid);"></button>
+    </div>
+    <script type="text/javascript" src="../Hover.js"></script>
```

Comment:
🤷 Could this be with the other script tags? Or above the divs if you want it scope to <body> (same for the LinearProgress html)

Reply:
I've made it so this can sit in the `<head>` now